### DISCUSSION TITLE
Please add Mode dev mainnet in chains

### DISCUSSION THREAD

**Discussion by flyingn1nja** [wevm/viem#1742] [General]
Could you please add in chains:
https://docs.mode.network/mode-developer-mainnet/mode-dev-mainnet-information

Mode testnet already supported:
```typescript
import { modeTestnet } from 'viem/chains'
```

So it would look like this:
```typescript
import { modeDevMainnet } from 'viem/chains'
```
or
```typescript
import { modeMainnet } from 'viem/chains'
```

Thank you in advance!

**Comment by tmm** [comment]
Feel free to add it yourself. Here are the contributing docs for [adding a chain.](https://github.com/wevm/viem/blob/main/.github/CONTRIBUTING.md#adding-a-chain)

**Comment by flyingn1nja** [comment]
@tmm  I did, it says permission denied. I can't push a branch.
